backend/src/database/repositories/dataQualityRepository.ts

```
@@ -0,0 +1,78 @@
+   */
+  static async findMembersWithTooManyIdentitiesPerPlatform(
+    options: IRepositoryOptions,
+    tenantId: string,
+    limit: number,
+    offset: number,
+    segmentId: string,
+  ) {
+    const qx = SequelizeRepository.getQueryExecutor(options)
+    return fetchMembersWithTooManyIdentitiesPerPlatform(qx, 1, tenantId, limit, offset, segmentId)
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)